eng/pipelines/templates/stages/archetype-rust-release.yml

```diff
@@ -0,0 +1,168 @@
+        #     parameters:
+        #       ArtifactPath: $(Pipeline.Workspace)/${{parameters.PipelineArtifactName}}
+        #       Artifacts: ${{parameters.Artifacts}}
+        #       ConfigFileDir: $(Pipeline.Workspace)/${{parameters.PipelineArtifactName}}/PackageInfo
+        #       MarkPackageAsShipped: true
+        #       ArtifactName: ${{parameters.PipelineArtifactName}}
+        #       SourceRootPath: $(Pipeline.Workspace)/azure-sdk-for-rust
+        #       PackageName: ${{artifact.name}}
+
+        - job: UpdatePackageVersion
+          displayName: "Update Package Versions"
+          condition: and(succeeded(), ne(variables['Skip.UpdatePackageVersion'], 'true'))
+          dependsOn: PublishPackage
+          steps:
+          - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+            parameters:
+              paths:
+              - "/*"
+
+          # Apply the version increment to each library, which updates the Cargo.toml and changelog files.
+          - task: PowerShell@2
+            displayName: Increment ${{artifact.name}} version
+            inputs:
+              targetType: filePath
+              filePath: $(Build.SourcesDirectory)/eng/scripts/Update-PackageVersion.ps1
+              arguments: >
+                -ServiceDirectory '${{parameters.ServiceDirectory}}'
+                -PackageName '${{artifact.name}}'
+
+          - template: /eng/common/pipelines/templates/steps/create-pull-request.yml
+            parameters:
+              PRBranchName: increment-package-version-${{parameters.ServiceDirectory}}-$(Build.BuildId)
+              CommitMsg: "Increment package version after release of ${{ artifact.name }}"
+              PRTitle: "Increment versions for ${{parameters.ServiceDirectory}} releases"
+              CloseAfterOpenForTesting: '${{parameters.TestPipeline}}'
+
+        - ${{ if eq(parameters.TestPipeline, true) }}:
+          - job: YankCrates
+            displayName: "Yank Crates"
+            dependsOn: PublishPackage
+            condition: and(succeeded(), ne(variables['Skip.PublishPackage'], 'true'))
+            steps:
+            - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+          
+            - download: current
+              displayName: Download ${{parameters.PipelineArtifactName}} artifact
+              artifact: ${{parameters.PipelineArtifactName}}
+              
+            - task: PowerShell@2
+              displayName: Yank Crates
+              inputs:
+                targetType: filePath
+                filePath: $(Build.SourcesDirectory)/eng/scripts/Yank-Crates.ps1
+                arguments:
+                  -CrateNames '${{artifact.name}}'
+                  -PackagesPath '$(Pipeline.Workspace)/${{parameters.PipelineArtifactName}}'
+                  -Token $(azure-sdk-cratesio-token)
+
+        # - job: PublishDocsMs
+
+        # - job: PublishDocs
```

Comment:
This shouldn't be necessary. Publishing to crates.io does it automatically e.g., https://github.com/heaths/ordinal-rs/blob/5b4254956cca64de22e5243a57e2686ace85c2e7/.github/workflows/release.yml#L38-L41

Reply:
excelent.  I'll remove the comments in following PR